src/tasks.js

```diff
@@ -0,0 +1,103 @@
+ko.tasks = (function () {
```

Comment:
This file includes snippets from other MIT-licensed projects, which is fine, but what do you think we should do about the licensing comments? The MIT license says you need to include a copy of the license if you use a "substantial portion" of the product, but never clarifies what "substantial" means or who adjudicates that. Other people who've discussed this seem to be risk-averse and suggest including the third-party license if you use any of their code at all.

Can we at least copy and past in the licenses for the two relevant projects as block comments here, inline in this file? Better still if they survive the minification process while remaining inline here in the code and not having to move to the top.